### PR TITLE
18-04-2022 vladnewfix3

### PR DESCRIPTION
**Pull request by thulsidk1**
this is the latest pull request (vladnewfix3)

### CODE REVIEW

File: resources.tf [hunk 1]
```diff
@@ -8,20 +8,35 @@ resource "aws_vpc" "vpc" {
...
 }
 
 resource "aws_eip" "ip" {
+  count                  = "${length(var.private_cidr)}"
+  vpc = true
+
+  tags = {
+    Name = "${var.environment_tag}-eip-private${count.index}"
+
+  }
```
Comment (vladimirmukhin) [inline resources.tf]:
```suggestion
    Name = "${var.environment_tag}-eip-private${count.index}"
  }
```

File: resources.tf [hunk 2]
```diff
@@ -8,20 +8,35 @@ resource "aws_vpc" "vpc" {
...
+}
+
+resource "aws_eip" "ip1" {
   vpc = true
+
+  tags = {
+    Name = "${var.environment_tag}-eip-public"
+
+  }
```
Comment (vladimirmukhin) [inline resources.tf]:
```suggestion
    Name = "${var.environment_tag}-eip-public"
  }
```

File: resources.tf [hunk 3]
```diff
@@ -83,7 +98,7 @@ resource "aws_route_table" "rtb_private1" {
   }
 
   tags = {
-    Name = var.environment_tag
+    Name = "${var.environment_tag}-rtb_private1${count.index}"
```
Comment (vladimirmukhin) [inline resources.tf]:
```suggestion
    Name = "${var.environment_tag}-rtb_private${count.index}"
```

File: resources.tf [hunk 4]
```diff
@@ -118,43 +133,38 @@ resource "aws_security_group" "sg_22" {
...
   ami                    = data.aws_ami.my_awslinux.id
   instance_type          = var.instance_type
-  subnet_id              = element(aws_subnet.subnet_public.*.id , count.index)
+  subnet_id              = aws_subnet.subnet_public[0].id
   key_name               = "lakshminarsimha"
   vpc_security_group_ids = [aws_security_group.sg_22.id]
 
   tags = {
-    Name = var.environment_tag
+    Name = "${var.environment_tag}-testInstance"
```
Comment (vladimirmukhin) [inline resources.tf]:
```suggestion
    Name = "${var.environment_tag}-public"
```

File: resources.tf [hunk 5]
```diff
@@ -118,43 +133,38 @@ resource "aws_security_group" "sg_22" {
...
   ami                    = data.aws_ami.my_awslinux.id
   instance_type          = var.instance_type
-  subnet_id              = element(aws_subnet.subnet_private1.*.id , count.index)
+  subnet_id              = aws_subnet.subnet_private1[0].id
   key_name               = "lakshminarsimha"
   vpc_security_group_ids = [aws_security_group.sg_22.id]
 
   tags = {
-    Name = var.environment_tag 
+    Name = "${var.environment_tag}-testInstance1"
```
Comment (vladimirmukhin) [inline resources.tf]:
```suggestion
    Name = "${var.environment_tag}-private"
```

File: variables.tf [hunk 6]
```diff
@@ -27,6 +27,12 @@ variable "availability_zone" {
   default     = "us-east-2a"
 }
 
+variable "availability_zone1" {
+  description = "availability zone to create subnet1"
+  default     = "us-east-2b"
+}
+
+
```
Comment (vladimirmukhin) [inline variables.tf]:
```suggestion

```

Comment (vladimirmukhin) [inline variables.tf]:
```suggestion

```

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

**Comment by thulsidk1** [comment]
close

**Pull request closed without merging**